code/modules/mob/living/life.dm

```diff
@@ -2,6 +2,9 @@
 	set invisibility = 0
 	set background = BACKGROUND_ENABLED
 
+	if(in_stasis && stat != CONSCIOUS)
```

Comment:
`if(in_stasis && stat)`... though I'd probably just check for if they're inside a cryobag instead of giving all relevant mobs a whole var just for this